betfairlightweight/endpoints/baseendpoint.py

```diff
@@ -101,8 +100,6 @@ def process_response(
 
         if lightweight:
             return result
-        elif self.client.lightweight and lightweight is not False:
```

Comment:
Why have you removed this? Without it would not return a lightweight response if the client has lightweight True

Reply:
I guess `lightweight` is guaranteed to be `not False` at this point, otherwise it would have returned in the line above already.